packages/api/endpoints/collections.js

```diff
@@ -197,16 +198,25 @@ async function put(req, res) {
   }
 
   try {
-    oldCollection = await collectionsModel.get({ name, version });
+    oldPgCollection = await collectionPgModel.get(knex, { name, version });
   } catch (error) {
     if (error.name !== 'RecordDoesNotExist') {
       throw error;
     }
     return res.boom.notFound(`Collection '${name}' version '${version}' not found`);
   }
 
+  try {
+    oldCollection = await collectionsModel.get({ name, version });
+  } catch (error) {
+    if (error.name !== 'RecordDoesNotExist') {
+      throw error;
+    }
+    log.warn(`Dynamo record for Collection '${name}' version '${version}' not found, proceeding to update with postgresql record alone`);
+  }
+
   collection.updatedAt = Date.now();
-  collection.createdAt = oldCollection.createdAt;
+  collection.createdAt = oldPgCollection.created_at.getTime();
```

Comment:
There is no conditional logic for collection writes, I think, so I'm wondering if the `translatePostgresCollectionToApiCollection` call before publishing to SNS is necessary

Reply:
I believe you are correct, good catch: 7a5e1ed96

Reply:
I guess ultimately once Dynamo goes away, we will need to do that. Not sure if it's worth keeping for that

Reply:
I'm not sure I understand-  are you suggesting the change in 7a5e1ed was not a good one?  If so, why?

Reply:
Yeah I'm contradicting myself because while we don't need `translatePostgresCollectionToApiCollection` for now, since we can just publish the updated Dynamo record to SNS, ultimately Dynamo goes away and we will have to translate the PG record to publish to SNS. But since we've already made the change, let's leave it for now